src/libs/ReportUtils.ts

```diff
@@ -2385,7 +2385,10 @@ function isWaitingForAssigneeToCompleteTask(report: OnyxEntry<Report>, parentRep
         return true;
     }
 
-    if (isOpenTaskReport(report, parentReportAction) && !report?.hasParentAccess && isReportManager(report)) {
+    if (isOpenTaskReport(report, parentReportAction) && isReportManager(report)) {
+        if (currentUserAccountID === report?.ownerAccountID) {
+            return !report?.hasParentAccess;
+        }
```

Comment:
Could you explain this changes?

Reply:
To display a green dot for parent reports only, but parent reports are visible only in the chat of the report's creator, so for other users, show the green dot on the report itself